app/services/db-export/compress-schema-folder.service.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+/**
+ * Creates a compressed tarball (.tgz) from a given schema folder
+ * @module CompressSchemaFolderService
+ */
+
+const tar = require('tar')
+
+/**
+ * Create a compressed tarball (.tgz) from a given schema folder
+ * @param {String} schemaFolderPath
+ *
+ * @returns {String} The path to the created tarball file
+ */
+async function go (schemaFolderPath) {
+  const file = `${schemaFolderPath}.tgz`
+
+  await tar.create(
+    {
+      gzip: true,
+      file
+    },
+    [schemaFolderPath]
+  )
+  return `${schemaFolderPath}.tgz`
```

Comment:
As we have defined `file` as a const we can just return that as the result rather than workout the file path again.

We also have a convention of putting a newline between the body of a method and its `return` statement.

```suggestion
  )
  
  return file
```